Packs/TOPdesk/Integrations/TOPdesk/TOPdesk.py

```diff
@@ -253,14 +253,15 @@ def attachment_upload(self, incident_id: Optional[str], incident_number: Optiona
 
         shutil.copyfile(demisto.getFilePath(file_entry)['path'], file_name)
         try:
-            with open(file_name, 'rb') as file_obj:
-                files = {'file': file_obj}
-                response = self._http_request(method='POST',
-                                              url_suffix=f"{endpoint}/attachments",
-                                              files=files,
-                                              data=request_params)
-        finally:
+            files = {'file': open(file_name, 'rb')}
```

Comment:
Any special reason you removed the with?

Reply:
Not sure why this has been changed. The original code for this should be the correct one, so with the 'with'